services/marathon/marathon.go

```diff
@@ -158,7 +180,7 @@ func createApps(tasksById map[string][]MarathonTask, marathonApps map[string]Mar
 
 		for _, task := range tasks {
 			if len(task.Ports) > 0 {
-				simpleTasks = append(simpleTasks, Task{Host: task.Host, Port: task.Ports[0]})
+				simpleTasks = append(simpleTasks, Task{Host: task.Host, Port: task.Ports[0], Alive: calculateTaskHealth(task.HealthCheckResults, marathonApps[appId].HealthChecks)})
```

Comment:
I like this approach, because I can actually eliminate the healthcheck path that only works for single check.